scripts/jenkins/cloud/manual/input.yml.example

```diff
@@ -31,6 +31,12 @@ cloud_env: ''
 # from an ephemeral machine, e.g. a docker container.
 local_ssh_key: false
 
+# If set to false, the Crowbar install steps will be skipped. This
```

Comment:
I think you also need to update the "local" scripts to add the new step/playbook.

Reply:
Done.